configs/maxtext/common.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 Google LLC
```

Comment:
For the path, I am having a [PR](https://github.com/GoogleCloudPlatform/ml-auto-solutions/pull/74) to add team subfolders under configs and dags. Inside of team subfolder, the code structure mainly depends on how your team organizes those tests. Probably no change is needed at this moment, and I will let you know once that's approved. Thanks!